src/main/java/net/atos/entng/support/controllers/TicketController.java

```diff
@@ -92,10 +91,10 @@ public class TicketController extends ControllerHelper {
     private final Storage storage;
     private final EventHelper eventHelper;
 
-    public TicketController(TicketServiceSql ts, TicketService ticketService, EscalationService es, UserService us, Storage storage) {
-        ticketServiceSql = ts;
-        this.ticketService = ticketService;
-        userService = us;
+    public TicketController(ServiceFactory serviceFactory, EscalationService es, Storage storage) {
+        ticketServiceSql = serviceFactory.ticketServiceSql();
+        this.ticketService = serviceFactory.ticketService();
+        userService = serviceFactory.userService();
         escalationService = es;
         this.storage = storage;
```

Comment:
escalationService = serviceFactory.getEscalationService();
this.storage = serviceFactory.getStorage();